### DISCUSSION TITLE
2.3 Standalone Install from ISO Fails

### DISCUSSION THREAD

**Discussion by trb143** [Security-Onion-Solutions/securityonion#2213] [Unsupported Versions] [answered]
Following on from #2022  I have installed a new version and only run the inital setup with all defaults.
This is from the ISO on a new VM with CPUs - 6 and 18GB Ram.

AT 78% all the containers were up and running (especially MySql) and there were no errors in the Log.

In the end, it failed and so-startus shows that: MySql was Missing and Strelka-base was in Error.  
The installer reported an error and did not complete.
I have left the machine in this state and uploaded the sosetup.log to "https://file.io/wjFgIIBhXYw2"

**Comment by weslambert** [comment]
Looks like the setup log is no longer there.  Are you still having issues with this?

**Reply by trb143** [reply]
Yes as this was created to show issues with MySQL.  https://file.io/SPAOD6vW7szV is a new copy of the log file.   
DefensiveDepth requested I try again.

**Reply by defensivedepth** [reply]
@trb143 Looks like it was a non-fatal error during the install - I would suggest restarting the system and  running `so-allow`, and then try logging in.

If you are seeing errors, run `sudo salt-call state.highstate` and post any errors you see.

**Comment by trb143** [comment]
Seems to be working fine on the restart following so-allow.  Fleet working fine.
so-status show so-strelka-backend in error but not sure which log file this is to get more info.

**Comment by dougburks** [comment, marked as answer]
The so-strelka-backend error is most likely a known issue which should be resolved in the next release:
https://github.com/Security-Onion-Solutions/securityonion/issues/2153
